Thing.h

```diff
@@ -723,17 +723,15 @@ class ThingDevice {
       typeJson.add(*type);
       type++;
     }
-
-    JsonArray links = descr.createNestedArray("links");
+    /*
+    JsonArray links = descr.createNestedArray("forms");
     {
       JsonObject links_prop = links.createNestedObject();
-      links_prop["rel"] = "properties";
       links_prop["href"] = "/things/" + this->id + "/properties";
```

Comment:
We can already be described in the TD the `/properties` please leave it and add as `op` member `['readallproperties', 'writeallproperties']`